activities/TankOp.activity/lib/tutorial.js

```diff
@@ -0,0 +1,86 @@
+define(["webL10n"], function (l10n) {
+	var tutorial = {};
+
+	tutorial.start = function() {
+		var steps = [
+			{
+				element: "",
+				orphan: true,
+				placement: "bottom",
+				title: l10n.get("TutoExplainTitle"),
+				content: l10n.get("TutoExplainContent")
+			},
+			{
+				element: "#fullscreen-button",
+				placement: "bottom",
+				title: l10n.get("TutoFullScreenButtonTitle"),
+				content: l10n.get("TutoFullScreenButtonContent")
```

Comment:
Do not mix indentation style in the same file

Reply:
Indention made uniform across tutorial.js file